src/events.js

```diff
@@ -3631,37 +3652,61 @@ export const populateReportManifestHeader = (currPage) => {
     document.getElementById('boxManifestCol1').appendChild(newDiv)
 }
 
-export const populateReportManifestTable = (currPage) => {
-    let currTable = document.getElementById('boxManifestTable');
-
+export const populateReportManifestTable = (currPage, searchSpecimenInstituteArray) => {
+    const currTable = document.getElementById('boxManifestTable');
```

Comment:
I don't think it needs to be null checked since the function calls are made after the inner HTML is created. 
https://github.com/episphere/biospecimen/pull/532/files#diff-0519669f950622c52f6e6514fbfa7441cbd9bd95c885b728cacb20e7a6a02b6fL82-L114